docker-compose.yml

```diff
@@ -29,7 +29,9 @@ services:
       echo "Waiting for materialized to start..." &&
       sleep 15 &&
       psql -h materialized -U mz_system -d materialize -p 6877 -c "ALTER SYSTEM SET max_clusters = 100;" &&
-      psql -h materialized -U mz_system -d materialize -p 6877 -c "ALTER SYSTEM SET max_sources = 100;"
+      psql -h materialized -U mz_system -d materialize -p 6877 -c "ALTER SYSTEM SET max_sources = 100;" &&
+      psql -h materialized -U mz_system -d materialize -p 6877 -c "ALTER SYSTEM SET max_aws_privatelink_connections = 10;" &&
+      psql -h materialized -U mz_system -d materialize -p 6877 -c "CREATE CONNECTION \"materialize\".\"public\".\"privatelink_conn\" TO AWS PRIVATELINK (AVAILABILITY ZONES = ('\''use1-az2'\'', '\''use1-az6'\''), SERVICE NAME = '\''com.amazonaws.us-east-1.materialize.example'\'');"
```

Comment:
Creating the privatelink via the init container rather than the TF tests, as if you were to drop the PL connection, the container crashes.